packages/gatsby-admin/package.json

```diff
@@ -28,6 +28,7 @@
     "lodash-es": "^4.17.15",
     "ncp": "^2.0.0",
     "nodemon": "^2.0.4",
+    "normalize.css": "^8.0.1",
```

Comment:
Side effect of cleaning up dependencies of `gatsby-recipes` - it had `normalize.css` that wasn't used there but apparently `gatsby-admin` used that, but didn't declare it ( failing CI without this addition - https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/50324/workflows/cc87edd1-381e-4663-a0a8-5cc8736c2b09/jobs/518517 )